CONTRIBUTIONS.md

```diff
@@ -0,0 +1,38 @@
+gti # Contribution Guidelines
+This is a community-driven, open source project that welcomes all contributions. Whether you're a seasoned contributor or new to the project, we're grateful for all contributions. 
+
+## Community standards 
+
+We are an inclusive community that values open dialogue, mutual respect, and fair treatment. Every submission will be treated equally and we encourage those with diverse backgrounds and perspectives to contribute. 
+
+We are part of the University of Massachusetts Amherst, so we adhere to the [UMass Code of Student Conduct](https://www.umass.edu/dean_students/codeofconduct).
+
+## Getting started 
+Before contributing to the project, take a look at the README file, which contains information about system requirements, environment setup steps, and a project summary. 
+
+Further documentation for this project is found in the docs folder. 
+
+## Selecting an issue
+Issues that are open for contribution are given the following labels:
+- good-first-issue
+  - Issues with this tag are suited for those that do not have previous experience with the project.
+- help-wanted
+  - Issues with this tag are open for contribution and are suited for those with experience in contribution. 
+
+## Submitting contributions
+
+To contribute to the project, do the following:
+- [Fork and clone](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the repository
+- Create a [branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository) for your issue 
+- Make a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) to the master repository 
```

Comment:
Instead of "master repository" here I would say "main branch of the upstream repository", since "master" is frequently used for branches instead and it could be confusing. You can check out https://stackoverflow.com/questions/9257533/what-is-the-difference-between-origin-and-upstream-on-github for more details on industry folks talk about different repositories.